src/RwFramework/GSyst.h

```diff
@@ -132,7 +132,7 @@ typedef enum EGSyst {
   kINukeTwkDial_FrInel_N,    ///< tweak inelastic       probability for nucleons, for given total rescattering probability
   kINukeTwkDial_FrAbs_N,     ///< tweak absorption      probability for nucleons, for given total rescattering probability
   kINukeTwkDial_FrPiProd_N,  ///< tweak pion production probability for nucleons, for given total rescattering probability
-
+  kINukehA2025_cex,
```

Comment:
I'm always very hesitant to have new enums added anywhere but at the end of the list.  Inserting them in the middle is problematic if the values are stored anywhere as it skews the numbers that might be saved elsewhere vs the new numbering.  It might be fine if the value is never stored, but that needs to be verified